lib/webdriver.py

```diff
@@ -121,35 +127,57 @@ def get_reservations(self, account_monitor: AccountMonitor) -> List[JSON]:
         # instead of requesting again later
         reservations = self._fetch_reservations(driver)
 
-        driver.quit()
+        self._quit_driver(driver)
         return reservations
 
     def _get_driver(self) -> Driver:
-        logger.debug("Starting webdriver for current session")
-        browser_path = self.checkin_scheduler.reservation_monitor.config.browser_path
+        # This environment variable is set in the Docker image
+        is_docker = os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1"
 
         driver_version = "mlatest"
-        if os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1":
-            # This environment variable is set in the Docker image. Makes sure a new driver
-            # is not downloaded as the Docker image already has the correct driver
+        if is_docker:
+            self._start_display()
+            # Make sure a new driver is not downloaded as the Docker image
+            # already has the correct driver
             driver_version = "keep"
 
+        logger.debug("Starting webdriver for current session")
+        browser_path = self.checkin_scheduler.reservation_monitor.config.browser_path
+
+        # Create a new temporary directory for Chrome profile
+        profile_dir = os.path.join("/tmp", "chrome_profile")
```

Comment:
Is this compatible with Windows? Specifically `/tmp`. It may be worth it to use https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir.

Reply:
Good catch, I will use, os.path.join(tempfile.gettempdir(), "chrome_profile")